activemodel/lib/active_model/serializers/json.rb

```diff
@@ -13,6 +13,90 @@ module JSON
         extend ActiveModel::Naming
 
         class_attribute :include_root_in_json, instance_writer: false, default: false
+        class_attribute :_to_json_formatter, instance_writer: false, default: KeyFormatter::Identity.new
+        class_attribute :_from_json_formatter, instance_writer: false, default: KeyFormatter::Identity.new
```

Comment:
Ideally, these would both be marked with # :nodoc:. Instead, I've prefixed them with `_`.

Does `# :nodoc:` work for attributes, or are there other ways to omit them from sdoc?